rs/registry/admin/src/main.rs

```diff
@@ -5530,6 +5551,129 @@ async fn propose_to_add_or_remove_node_provider(
     };
 }
 
+async fn propose_to_install_code(
+    cmd: ProposeToChangeNnsCanisterCmd,
+    agent: Agent,
+    proposer: NeuronId,
+) {
+    let canister_client = GovernanceCanisterClient(NnsCanisterClient::new(
+        agent,
+        GOVERNANCE_CANISTER_ID,
+        Some(proposer),
+    ));
+
+    let canister_id = Some(PrincipalId::try_from(cmd.canister_id).unwrap());
```

Comment:
I think you can just use from instead of try_from?

Reply:
Done.